app/src/main/java/com/github/swent/echo/compose/navigation/AppNavigationHost.kt

```diff
@@ -110,3 +119,10 @@ fun AppNavigationHost(
         }
     }
 }
+
+/** Quit the app when the hardware back button is pressed */
+@Composable
+fun QuitAppOnHardwareBackButtonPressPress() {
```

Comment:
I don't know if it's a small typo but you wrote 2 times "...PressPress" in the function name